src/app/shared/horizontal-carousel/horizontal-carousel.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {HorizontalCarousel, HorizontalCarouselModule} from './horizontal-carousel';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {DocsAppTestingModule} from '../../testing/testing-module';
+
+describe('HorizontalCarousel', () => {
+  let fixture: ComponentFixture<HorizontalCarousel>;
+  let component: HorizontalCarousel;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule(
+      {imports: [HorizontalCarouselModule, DocsAppTestingModule]}
+    ).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(HorizontalCarousel);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should not show prev nav arrow', () => {
+    const navPrevious = fixture.nativeElement.querySelector('.docs-carousel-nav-prev');
+    expect(navPrevious).toBeNull();
+  });
+
+  it('should show prev nav arrow after increasing index', () => {
+    component.next();
+    expect(component.index).toEqual(1);
+
+    const navPrevious = fixture.nativeElement.querySelector('.docs-carousel-nav-prev');
+    expect(navPrevious).toBeDefined();
+  });
+
+  it('should trigger onResize method when window is resized', () => {
+    const spyOnResize = spyOn(component, 'onResize');
+    window.dispatchEvent(new Event('resize'));
+    expect(spyOnResize).toHaveBeenCalled();
```

Comment:
Hmm, on its own, I don't think this test has much value. I think it would be good to combine it with the one below and add the desired data to the event. Additionally, I think you can then make `resizeCarousel` private